CHANGELOG.md

```diff
@@ -35,6 +35,11 @@
 * Add `private_only` configuration to `prefixed_toplevel_constant` rule  
   [Keith Smiley](https://github.com/keith)
   [#2315](https://github.com/realm/SwiftLint/pull/2315)
+  
+* Make rule `explicit_type_interface` compatible with rule
```

Comment:
can you please mention that this is possible with the `allow_redundancy` configuration key?

Reply:
Done.